### PR TITLE
Fix "whitespace" words - docs/framework

### PR DESCRIPTION
**Pull request by lucasnobredev**
## Fix "whitespace" words - docs/framework

This PR came out from this [issue](https://github.com/dotnet/docs/issues/4439). It's the third PR and it's all about docs/framework.

## Brief explanation
We should use **white-space** for adjectives and **white space** for nouns.

## Suggested Reviewers
@mairaw @rpetrusha @wstaelens

### CODE REVIEW

File: docs/framework/xaml-services/xaml-related-clr-attributes-for-custom-types-and-libraries.md [hunk 1]
```diff
@@ -163,7 +163,7 @@ This topic describes the common language runtime (CLR) attributes that are defin
   
  **Arguments:** None.  
   
- <xref:System.Windows.Markup.WhitespaceSignificantCollectionAttribute> indicates that a collection type should be processed as whitespace significant by a XAML processor, which influences the construction of the XAML node stream's value nodes within the collection. For more information, see [Whitespace Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
+ <xref:System.Windows.Markup.WhitespaceSignificantCollectionAttribute> indicates that a collection type should be processed as white-space significant by a XAML processor, which influences the construction of the XAML node stream's value nodes within the collection. For more information, see [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).
```
Comment (rpetrusha) [inline docs/framework/xaml-services/xaml-related-clr-attributes-for-custom-types-and-libraries.md]: White space Processing --> White-space processing

File: docs/framework/xaml-services/xml-character-entities-and-xaml.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ XAML uses character entities defined in XML for special characters. This topic d
...
 |(numeric character mappings)|&#*[integer]*; or &#x*[hex]*;|XAML supports numeric character mappings into the encoding that is active.|  
-|(nonbreaking space)|&\#160; (assuming UTF-8 encoding)|For flow document elements, or elements that take text such as the WPF <xref:System.Windows.Controls.TextBox>, nonbreaking spaces are not normalized out of the markup, even for `xml:space="default"`. (For more information, see [Whitespace Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).)|  
+|(nonbreaking space)|&\#160; (assuming UTF-8 encoding)|For flow document elements, or elements that take text such as the WPF <xref:System.Windows.Controls.TextBox>, nonbreaking spaces are not normalized out of the markup, even for `xml:space="default"`. (For more information, see [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).)|
```
Comment (rpetrusha) [inline docs/framework/xaml-services/xml-character-entities-and-xaml.md]: White space Processing --> White-space processing

File: docs/framework/xaml-services/xml-character-entities-and-xaml.md [hunk 3]
```diff
@@ -60,4 +60,4 @@ XAML uses character entities defined in XML for special characters. This topic d
  [XAML Overview (WPF)](../../../docs/framework/wpf/advanced/xaml-overview-wpf.md)  
  [Markup Extensions and WPF XAML](../../../docs/framework/wpf/advanced/markup-extensions-and-wpf-xaml.md)  
  [XamlName Grammar](../../../docs/framework/xaml-services/xamlname-grammar.md)  
- [Whitespace Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md)
+ [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md)
```
Comment (rpetrusha) [inline docs/framework/xaml-services/xml-character-entities-and-xaml.md]: White space Processing --> White-space processing

File: docs/framework/xaml-services/xml-space-handling-in-xaml.md [hunk 4]
```diff
@@ -3,13 +3,13 @@ title: "xml:space Handling in XAML"
 ms.date: "03/30/2017"
 helpviewer_keywords: 
   - "XAML [XAML Services], xml:space attribute"
-  - "XAML [XAML Services], whitespace processing"
+  - "XAML [XAML Services], white space processing"
```
Comment (rpetrusha) [inline docs/framework/xaml-services/xml-space-handling-in-xaml.md]: white space --> white-space

File: docs/framework/xaml-services/xml-space-handling-in-xaml.md [hunk 5]
```diff
@@ -3,13 +3,13 @@ title: "xml:space Handling in XAML"
...
   - "xml:space attribute [XAML Services]"
-  - "whitespace processing [XAML Services]"
+  - "white space processing [XAML Services]"
```
Comment (rpetrusha) [inline docs/framework/xaml-services/xml-space-handling-in-xaml.md]: white space --> white-space

File: docs/framework/xaml-services/xml-space-handling-in-xaml.md [hunk 6]
```diff
@@ -3,13 +3,13 @@ title: "xml:space Handling in XAML"
...
 ms.assetid: 5e1814f0-5b30-43d5-8c88-dede335a89d7
 ---
 # xml:space Handling in XAML
-The `xml:space` attribute is an XML-defined attribute that declares the significant whitespace processing behavior within an object element. This behavior is relevant for all content (inner text) contained within the element where `xml:space` is declared, and also scopes to child elements.  
+The `xml:space` attribute is an XML-defined attribute that declares the significant white space processing behavior within an object element. This behavior is relevant for all content (inner text) contained within the element where `xml:space` is declared, and also scopes to child elements.
```
Comment (rpetrusha) [inline docs/framework/xaml-services/xml-space-handling-in-xaml.md]: white space processing --> white-space processing

File: docs/framework/xaml-services/xml-space-handling-in-xaml.md [hunk 7]
```diff
@@ -26,14 +26,14 @@ The `xml:space` attribute is an XML-defined attribute that declares the signific
 ## Remarks  
  The definition for the `xml:space` attribute in XAML including its two possible values is derived from `xml:space` as defined as a "special attribute" by W3C specifications for XML.  
   
- The default value of the `xml:space` attribute is the literal value `"default"`. For the value `"default"`, or if `xml:space` is not indicated at all, the behavior of significant whitespace parsing is the default handling, as defined in the topic [Whitespace Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
+ The default value of the `xml:space` attribute is the literal value `"default"`. For the value `"default"`, or if `xml:space` is not indicated at all, the behavior of significant white space parsing is the default handling, as defined in the topic [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).
```
Comment (rpetrusha) [inline docs/framework/xaml-services/xml-space-handling-in-xaml.md]: white space parsing --> white-space parsing
White space Processing --> White space processing

File: docs/framework/xaml-services/xml-space-handling-in-xaml.md [hunk 8]
```diff
@@ -26,14 +26,14 @@ The `xml:space` attribute is an XML-defined attribute that declares the signific
...
+ The default value of the `xml:space` attribute is the literal value `"default"`. For the value `"default"`, or if `xml:space` is not indicated at all, the behavior of significant white space parsing is the default handling, as defined in the topic [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
   
- To preserve whitespace within object element content, specify `xml:space="preserve"` on that object element.  
+ To preserve white space within object element content, specify `xml:space="preserve"` on that object element.  
   
  Under most interpretations, the `xml:space` attribute effects and the value of the attribute are scoped to child elements.  
   
- For a complete discussion of whitespace processing in XAML, see [Whitespace Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
+ For a complete discussion of white space processing in XAML, see [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).
```
Comment (rpetrusha) [inline docs/framework/xaml-services/xml-space-handling-in-xaml.md]: white space processing --> white-space processing
in the title: White space Processing --> White-space processing

File: docs/framework/xaml-services/xml-space-handling-in-xaml.md [hunk 9]
```diff
@@ -26,14 +26,14 @@ The `xml:space` attribute is an XML-defined attribute that declares the signific
...
+ To preserve white space within object element content, specify `xml:space="preserve"` on that object element.  
   
  Under most interpretations, the `xml:space` attribute effects and the value of the attribute are scoped to child elements.  
   
- For a complete discussion of whitespace processing in XAML, see [Whitespace Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
+ For a complete discussion of white space processing in XAML, see [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md).  
   
 ## See Also  
- [Whitespace Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md)  
+ [White space Processing in XAML](../../../docs/framework/xaml-services/whitespace-processing-in-xaml.md)
```
Comment (rpetrusha) [inline docs/framework/xaml-services/xml-space-handling-in-xaml.md]: White space Processing --> White-space processing

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thank you so much, @lucasnobredev, for continuing your work on correcting the use of white-space and whitespace; we really appreciate it. I've noted a number of cases where "white space" should be "white space" and also noted some changes that I'd appreciate your making, even though they're unrelated to your PR. I also changed some formatting problems introduced by our migration from our old authoring system to GitHub.

**Comment by lucasnobredev** [comment]
Done!

**Review by rpetrusha — APPROVED** [review]
Thanks for making the additional changes, @lucasnobredev, and thanks for your commitment to this project. I've made a few additional changes. This is now ready to merge. Changes should go live on docs.microsoft.com in the next day or two.

**Pull request merged**